### DISCUSSION TITLE
Max temp set to 260

### DISCUSSION THREAD

**Discussion by mattranieri** [Jyers/Marlin#490] [Q&A] [answered]
I like to print my ASA at 265 degrees but I noticed this firmware limits the printer to 260.
printing at 260 generates a thermal runaway error and stops the print after about 1 minutes.
I am assuming its because the nozzle may creep over 260 during a cycle and that triggers it.
Can this please be fixed on the next revision?

**Comment by aryajuanda** [comment, marked as answer]
you can build the firmware manually and set your max temp to higher temp.. 
default is 275 with -15 (overshoot) so max available is 260 is max safe temperature for stock ender3 v2 without any mod.. is not recommend to set it higher by default..

#define HEATER_0_MAXTEMP 275 <-- in the Configuration.h
